packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -141,24 +146,34 @@ export const getPublicEvent = async (username: string, eventSlug: string, prisma
     };
   }
 
+  const usersOrTeamQuery = isTeamEvent
```

Comment:
If it's a team event we now only query for the team, instead of running the OR query. Left the OR in place for all other cases, so team is even then still used as a fallback (for example in case a user in the future will use this as atom, and forget to pass in the `isTeamEvent` boolean.

Reply:
Yes you are right, I've just now reverted this decision to NOT always fallback to the team event. The `isTeamEvent` boolean is not required though. So if you don't pass it in, it defaults to a user event and won't fetch the team event as fallback anymore.